File: Modul 4/Soal 5/PRAK305-2210817120012-Noorlaini Marratain.c
#include <stdio.h>

int main()
{
    int d,h,s,hk,m;
    scanf("%d", &hk);
    d=hk/84600;
    h=(hk/3600)%24;
    m=(hk/60)%60;
    s=hk%60;
    if(hk<=3600){
        printf("%.2d:%.2d:%.2d\n",h,m,s);}
    else if(hk>3600 && hk<84600){
        printf("%.2d:%.2d:%.2d\n",h,m,s);}
    else if(hk>=84600){
        printf("%d hari %.2d:%.2d:%.2d\n",d,h,m,s);}
}

File: Modul 4/Soal 5/PRAK305-2210817120012-Noorlaini Marratain.py
from ast import Break


hk=int(input())
d=int(hk/84600)
h=int((hk/3600)%24)
m=int((hk/60)%60)
s=int(hk%60)
if(hk<=3600):
    print("%.2d:%.2d:%.2d\n" % (h,m,s))
elif(hk>=3600 and hk<84600):
    print("%.2d:%.2d:%.2d\n" % (h,m,s))
elif(hk>=84600):
    print("%d hari %.2d:%.2d:%.2d\n" % (d,h,m,s))
if (hk>4) :
    Break